src/models.rs

```diff
@@ -90,32 +90,45 @@ pub struct VrfRequest {
     pub data_id: String,
 }
 
-
 #[derive(Queryable, Debug, QueryableByName, Selectable)]
 #[diesel(primary_key(data_id))]
 #[diesel(check_for_backend(diesel::pg::Pg))]
 #[diesel(table_name = crate::schema::oo_requests)]
 pub struct OORequest {
-    pub network: String, 
-    pub data_id: String, 
-    pub assertion_id: String, 
-    pub domain_id: String, 
-    pub claim: String, 
-    pub asserter: String, 
-    pub disputer: Option<String>, 
-    pub disputed: Option<bool>, 
+    pub network: String,
+    pub data_id: String,
+    pub assertion_id: String,
+    pub domain_id: String,
+    pub claim: String,
+    pub asserter: String,
+    pub disputer: Option<String>,
+    pub disputed: Option<bool>,
     pub dispute_id: Option<String>,
-    pub callback_recipient: String, 
-    pub escalation_manager: String, 
-    pub caller: String, 
-    pub expiration_timestamp: NaiveDateTime, 
+    pub callback_recipient: String,
+    pub escalation_manager: String,
+    pub caller: String,
+    pub expiration_timestamp: NaiveDateTime,
     pub settled: Option<bool>,
-    pub settlement_resolution: Option<bool>, 
-    pub settle_caller: Option<String>, 
-    pub currency: String, 
-    pub bond: BigDecimal, 
-    pub _cursor:  (Bound<i64>, Bound<i64>),
-    pub identifier: String, 
+    pub settlement_resolution: Option<bool>,
+    pub settle_caller: Option<String>,
+    pub currency: String,
+    pub bond: BigDecimal,
+    pub _cursor: (Bound<i64>, Bound<i64>),
+    pub identifier: String,
     pub updated_at: NaiveDateTime,
     pub updated_at_tx: String,
-}
\ No newline at end of file
+}
+
+#[derive(Debug, Queryable, Selectable, QueryableByName)]
+#[diesel(table_name = crate::schema::pragma_devnet_dispatch_event)]
+#[diesel(check_for_backend(diesel::pg::Pg))]
+pub struct PragmaDevnetDispatchEvent {
+    pub network: String,
+    pub block_hash: String,
+    pub block_number: i64,
+    pub block_timestamp: NaiveDateTime,
+    pub transaction_hash: String,
+    pub hyperlane_message_nonce: BigDecimal,
+    pub feeds_updated: Option<Vec<String>>,
+    pub _cursor: (Bound<i64>, Bound<i64>),
```

Comment:
cursor shouldn't be a range in that case no?

Reply:
ah yeah u're right - updated